eng/pipelines/jobs/bash-build.yml

```diff
@@ -40,8 +40,8 @@ jobs:
       ${{ if ne(parameters.name, 'FreeBSD_x64')}}:
         RunArguments: >-
           docker run --privileged --rm
-          -v "$(Build.SourcesDirectory):/root/coresetup"
-          -w="/root/coresetup"
+          -v "$(Build.SourcesDirectory):/root/runtime"
```

Comment:
don't we need a condition here?

Reply:
We don't need a condition here. This change (as well as the other similar changes) are mainly cosmetic. This argument to docker says "map the $(Build.SourcesDirectory) to the directory /root/runtime in the docker container". We could delay this particular change until after consolidation, but I felt that it fits better in this PR.